test/testdata/failing-dc.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       intervalSeconds: 1
       maxSurge: 25%
       maxUnavailable: 25%
-      timeoutSeconds: 5
+      timeoutSeconds: 40
```

Comment:
any follow-up here? Possibly hiding a problem.

Reply:
> any follow-up here? Possibly hiding a problem.

noted in commit message.  Yes it would be.